src/commands/cloud-run/cli.ts

```diff
@@ -1,3 +1,4 @@
 import {CloudRunFlareCommand} from './flare'
+import {InstrumentCommand} from './instrument'
 
-module.exports = [CloudRunFlareCommand]
+module.exports = [InstrumentCommand, CloudRunFlareCommand]
```

Comment:
Since the command is a WIP, let's not add it to the list here. This way it won't be exposed to users